pytest_html/plugin.py

```diff
@@ -55,7 +56,8 @@ def pytest_configure(config):
     htmlpath = config.option.htmlpath
     # prevent opening htmlpath on slave nodes (xdist)
     if htmlpath and not hasattr(config, 'slaveinput'):
-        config._html = HTMLReport(htmlpath)
+        contained_html = config.getoption('self_contained_html')
+        config._html = HTMLReport(htmlpath, contained_html=contained_html)
```

Comment:
I would call this variable and option `self_contained` as it's an attribute of the `HTMLReport`.